domains/client/domain-operator/src/fraud_proof.rs

```diff
@@ -151,24 +151,30 @@ where
         _local_receipt: &ExecutionReceiptFor<Block, CBlock>,
         mismatch_type: BundleMismatchType,
         bundle_index: u32,
-        _bad_receipt_hash: H256,
+        bad_receipt_hash: H256,
     ) -> Result<FraudProof<NumberFor<PCB>, PCB::Hash>, FraudProofError>
     where
         PCB: BlockT,
     {
         match mismatch_type {
             // TODO: Generate a proper proof once fields are in place
             BundleMismatchType::TrueInvalid(_invalid_type) => Ok(FraudProof::InvalidBundles(
-                InvalidBundlesFraudProof::ValidAsInvalid(ValidAsInvalidBundleEntryFraudProof::new(
+                InvalidBundlesFraudProof::TrueInvalid(TrueInvalidBundleEntryFraudProof::new(
+                    bad_receipt_hash,
                     domain_id,
                     bundle_index,
+                    0,
```

Comment:
Why hardcode it to 0? the extrinsic index should come from `invalid_type`

Reply:
@NingLin-P Fraud proof generation is unimplemented hence the default values. Reason why it is not `_ => unimplemented!()` is because this function is being called on main. So, I do not want to panic here as it would crash the node.